docs/kyma/04-05-use-your-own-domain.md

```diff
@@ -7,116 +7,197 @@ This guide explains how to deploy Kyma on a cluster using your own domain.
 
 >**TIP:** Get a free domain for your cluster using services like [freenom.com](https://www.freenom.com) or similar.
 
-Choose your cloud provider and get started:
+## Prerequisites
 
-<div tabs name="provider-domain">
+<div tabs name="prerequisites" group="use-your-own-domain">
   <details>
-  <summary>
+  <summary label="GKE">
   GKE
   </summary>
 
-
-Install Kyma on a [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) cluster.
-
-## Prerequisites
+- A domain for your [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) cluster
 - [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project with Kubernetes Engine API enabled
-- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.13.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
 - [gcloud](https://cloud.google.com/sdk/gcloud/)
 - [wget](https://www.gnu.org/software/wget/)
-- A domain for your GKE cluster
 
->**NOTE:** Running Kyma on GKE requires three [`n1-standard-4` machines](https://cloud.google.com/compute/docs/machine-types). You create these machines when you complete the **Prepare the GKE cluster** step.
+>**NOTE:** Running Kyma on GKE requires three [`n1-standard-4` machines](https://cloud.google.com/compute/docs/machine-types). You create these machines when you complete the **Prepare the cluster** step.
+
+  </details>
+  <details>
+  <summary label="AKS">
+  AKS
+  </summary>
+
+- A domain for your [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) (AKS) cluster
+- [Microsoft Azure](https://azure.microsoft.com)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
+- Tiller 2.10.0 or higher
+- [Docker](https://www.docker.com/)
+- A [Docker Hub](https://hub.docker.com/) account
+- [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+>**NOTE:** Running Kyma on AKS requires three [`Standard_D4_v3` machines](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-1). You create these machines when you complete the **Prepare the AKS cluster** step.
```

Comment:
the same thing about **Prepare the AKS cluster** as above

but in general, I **really** like the new layout, pico bello 👍 

<img src="https://user-images.githubusercontent.com/17568639/65416951-3a868380-ddf9-11e9-81ea-ff225a92e74d.jpg" data-canonical-src="https://user-images.githubusercontent.com/17568639/65416951-3a868380-ddf9-11e9-81ea-ff225a92e74d.jpg" width="300" height="300" />